# Spotify Top 50 Musics in 2025 - Outliers e Casos Fora do Padrão

### 1. Carregando o dataset

In [1]:
import pandas as pd

#index_col=0 impede que o pandas crie uma coluna extra de índice
df = pd.read_csv('2025_top50_songs.csv', index_col=0)
df.head()

,song_title,artist,streams_2025_billions,primary_genre,bpm,duration_seconds,release_year,artist_country,explicit,danceability,energy,valence,acousticness,peak_global_chart_position,dataset_part
wrapped_2025_rank,,,,,,,,,,,,,,,
1,Die With A Smile,Lady Gaga & Bruno Mars,1.70,Pop,120,251,2024,USA,False,0.72,0.68,0.55,0.18,1,Spotify Wrapped 2025 Top 50 Global Songs
2,APT.,ROSE & Bruno Mars,1.62,K-Pop/Pop,145,173,2024,South Korea,False,0.86,0.79,0.88,0.05,1,Spotify Wrapped 2025 Top 50 Global Songs
3,Espresso,Sabrina Carpenter,1.55,Pop,104,175,2024,USA,False,0.80,0.73,0.96,0.06,1,Spotify Wrapped 2025 Top 50 Global Songs
4,Please Please Please,Sabrina Carpenter,1.48,Pop,107,186,2024,USA,False,0.70,0.66,0.87,0.10,1,Spotify Wrapped 2025 Top 50 Global Songs
5,Taste,Sabrina Carpenter,1.41,Pop,117,177,2024,USA,True,0.78,0.70,0.82,0.08,1,Spotify Wrapped 2025 Top 50 Global Songs


###2. Música mais antiga do ranking

In [2]:
antigas = df.loc[df['release_year'] < 2000]
antigas[['song_title', 'artist', 'release_year', 'streams_2025_billions', 'peak_global_chart_position']]

,song_title,artist,release_year,streams_2025_billions,peak_global_chart_position
wrapped_2025_rank,,,,,
44,Running Up That Hill,Kate Bush,1985,0.62,3


### 3. Músicas com menos streams do ranking

In [3]:
df.sort_values('streams_2025_billions',).head(5)

,song_title,artist,streams_2025_billions,primary_genre,bpm,duration_seconds,release_year,artist_country,explicit,danceability,energy,valence,acousticness,peak_global_chart_position,dataset_part
wrapped_2025_rank,,,,,,,,,,,,,,,
50,Azizam,Ed Sheeran,0.56,Pop/Folk,88,196,2023,UK,False,0.70,0.52,0.72,0.32,3,Spotify Wrapped 2025 Top 50 Global Songs
49,Bodyguard,Beyoncé,0.57,Country,90,218,2024,USA,False,0.60,0.56,0.66,0.38,4,Spotify Wrapped 2025 Top 50 Global Songs
48,That's So True,Gracie Abrams,0.58,Indie Pop,97,185,2024,USA,False,0.64,0.55,0.66,0.22,1,Spotify Wrapped 2025 Top 50 Global Songs
47,Espresso (Remix),Sabrina Carpenter & Jimin,0.59,Pop/K-Pop,104,178,2024,USA/S.Korea,False,0.82,0.74,0.92,0.05,2,Spotify Wrapped 2025 Top 50 Global Songs
46,Shake It Off,Taylor Swift,0.60,Pop,160,219,2014,USA,False,0.65,0.80,0.94,0.06,1,Spotify Wrapped 2025 Top 50 Global Songs


### 4. Caso Clairo

In [4]:
clairo = df.loc[df['artist'] == 'Clairo']
clairo[['song_title', 'streams_2025_billions', 'peak_global_chart_position', 'danceability', 'energy', 'valence']]

,song_title,streams_2025_billions,peak_global_chart_position,danceability,energy,valence
wrapped_2025_rank,,,,,,
43,Pretty Girl,0.63,22,0.7,0.52,0.68


### 5. Distribuição geral de streams

In [5]:
df.sort_values('streams_2025_billions', ascending=False)[['song_title', 'artist', 'streams_2025_billions']]

,song_title,artist,streams_2025_billions
wrapped_2025_rank,,,
1,Die With A Smile,Lady Gaga & Bruno Mars,1.70
2,APT.,ROSE & Bruno Mars,1.62
3,Espresso,Sabrina Carpenter,1.55
4,Please Please Please,Sabrina Carpenter,1.48
5,Taste,Sabrina Carpenter,1.41
6,Birds Of A Feather,Billie Eilish,1.38
7,Not Like Us,Kendrick Lamar,1.35
8,"Good Luck, Babe!",Chappell Roan,1.29
9,Beautiful Things,Benson Boone,1.24


### 6. Conclusões

* O caso mais surpreendente do ranking é **Kate Bush** com "Running Up That Hill",
  lançada em **1985** e presente no top 50 de 2025 com **0.62 bilhões** de streams.
  A música voltou às paradas globais após aparecer em Stranger Things em 2022, e novamente em 2025,
  provando que cultura pop e streaming podem ressuscitar músicas décadas depois
  do lançamento.

* A diferença entre o primeiro e o último lugar em streams é de apenas **1.14
  bilhões** (1.70 vs 0.56). Para um top 50 global, isso mostra que o piso de
  entrada é alto, qualquer música aqui já é um fenômeno por si só.

* **Clairo** é o outlier mais curioso da lista: posição **22** no chart global,
  a mais baixa do ranking inteiro, com apenas **0.63 bilhões** de streams e
  perfil sonoro bem diferente da média, baixa energia (0.52) e danceability
  (0.41). Ela chegou aqui não pelo impacto imediato, mas por uma base de fãs
  fiel que consome de forma consistente.

* O bottom 5 em streams tem nomes como **Shake It Off** (Taylor Swift) e
  **Espresso Remix** (Sabrina Carpenter), mostrando que até as "menores"
  músicas do ranking pertencem a artistas dominantes em 2025.